HOWTO_RELEASE.md

```diff
@@ -0,0 +1,14 @@
+Release procedure is as follows:
+
+- Let $vmaj, $vmin and $vpat be Major, Minor and Patch version numbers
+- Drop "dev" suffix from $vpat
+- Let $ver be "$vmaj.$vmin.$vpat"
+- Make sure Version variable is set correctly in main.go ($ver)
+- Compile CHANGELOG.md section for $ver
+  (please only include user-relevant changes, and be concise)
+- `git tag -a v$ver`
```

Comment:
This should be after the branching section

Reply:
It would make no difference, no  ? I like to idea of tagging *before* branching so that `git-describe` counts from latest release...

Reply:
Humm, that kinda makes sense 🤔